app/controllers/agent/ManageAgentForms.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.agent
+
+import form.EnumMapping
+import models.propertyrepresentation.ManageAgentOptions
+import play.api.data.{Form, Forms}
+
+object ManageAgentForms {
+
+  val manageAgent: Form[ManageAgentOptions] = Form(Forms.single("manageAgent" -> EnumMapping(ManageAgentOptions)))
```

Comment:
is this a form for ManageAgentForms or ManageAgentOptions? the singleton object for this form has me confused.

Reply:
ManageAgentForms is meant to be deleted, it is not used anymore. It is part of the refactor i did this. I forgot to remove it

Reply:
im using ManageAgentRequest instead